draft-ietf-tls-esni.md

```diff
@@ -353,15 +403,17 @@ ClientHello. This extension is defined as follows:
 ~~~~
 
 nonce
-: A 16-byte nonce exported from the HPKE encryption context. See {{send-ech}}
-for details about its computation.
-
-Finally, requirements in {{client-behavior}} and {{server-behavior}} require
-implementations to track, alongside each PSK established by a previous
-connection, whether the connection negotiated this extension with the
-"ech_accept" response type. If so, this is referred to as an "ECH PSK".
-Otherwise, it is a "non-ECH PSK". This may be implemented by adding a new field
-to client and server session states.
+: A 16-byte nonce exported from the HPKE encryption context, as described in
+{{send-ech}}.
+
+Per {{client-behavior}} and {{server-behavior}}, implementations are required to
+track, alongside each PSK established by a previous connection, whether the
+connection negotiated this extension with the "ech_accept" response type. If so,
+this is referred to as an "ECH PSK". Otherwise, it is a "non-ECH PSK". This may
+be implemented by adding a new field to client and server session states.
+[[TODO: Why does discussion of the PSK belong in this section, which is about
```

Comment:
It probably doesn't. Can you find a better home for it?

Reply:
Not until I understand the PSK story. I'll leave this unresolved for now.

Reply:
Let's remove the TODO then.

Reply:
Done.